tests/integrations/protocol/test_offseter.cairo

```diff
@@ -13,6 +13,7 @@ from tests.integrations.protocol.library import (
     admin_instance as admin,
     anyone_instance as anyone,
     carbonable_offseter_instance as offseter,
+    carbonable_project_instance as project,
```

Comment:
Comment for the file :

Should we add an integration test where we change the value of absorption, when a forest is burning ?
![Burning-forest](https://user-images.githubusercontent.com/3840436/210594840-8250fdec-add0-40ac-a60e-c48c0ba3a459.PNG)
st burn.

Reply:
I don't think it will increase the coverage of the tests, and this test will take times to set up and to be run imo.

Reply:
good for me in this case 👍